src/ui/osx/TogglDesktop/test2/NSTextField+Ext.m

```diff
@@ -0,0 +1,22 @@
+//
+//  NSTextField+Ext.m
+//  TogglDesktop
+//
+//  Created by Nghia Tran on 11/20/18.
+//  Copyright © 2018 Alari. All rights reserved.
+//
+
+#import "NSTextField+Ext.h"
+
+@implementation NSTextField(Extension)
+
+-(void)resetCursorColor
+{
+    NSTextView *textField = (NSTextView *)[self currentEditor];
```

Comment:
I'd use a better name for this variable. `editor` or `textView` or something like that. `textField` seems out of place here.

Reply:
Yup, it's my mistake 👍